src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -20,7 +20,7 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the applicants identified by the index numbers used in the displayed person list.\n"
-            + "Parameters: INDEX... (must be positive integers)\n"
+            + "Parameters: INDEX...\n"
```

Comment:
Why not keep the (must be positive integers) in?

Reply:
Ah because this message would only come up if they didn't input a positive integer, but I appended the error message from ParserUtil that would tell the user if they did not input a positive integer. So if I leave this in, it would tell the user twice. So might as well take out 😅